### DISCUSSION TITLE
Elevation of 3D tile layer

### DISCUSSION THREAD

**Discussion by dschor-cubicsearch** [visgl/deck.gl#7944] [Q&A]
Is there a way to get the elevation data for the google 3D tile layer at a point?

Also is there a way to get the extruded geojson elevation over the 3D tile layer when is using terrain extension?

**Comment by Pessimistress** [comment]
If it's visible you can call the [pickObject](https://deck.gl/docs/api-reference/core/deck#pickobject) method with `unproject3D: true`

**Reply by dschor-cubicsearch** [reply]
I tried this.  You can click on the object to get the x,y of the pixel.  It worked with some areas, but I'm trying to do this in an array of Points.  Is there a way to get the x,y of all rendered Points in a geojson layer?  Or is there a simple way to get the mesh/Tile3D elevation from Latitude and Longitude?  Lat and Long is most useful, and if I have to pick, I'd like to have a way to pick from an array of x,y points, but I can't find a way to get to the Layer's array of x,y for the points that I added.

**Comment by harshalpatil7372** [comment]
i was also looking for the same option where by passing lat log we can get elevation value of that point in 3d tiles
